active/Expander/Expander.md

```diff
@@ -0,0 +1,155 @@
+| :---------- | :------- | :------- |
+| ExpandDirection | Sets the direction of expansion | Down = 0 |
+| IsExpanded | Whether or not control is expanded | False |
+
+### Events
+| Name | Description | 
+| :---------- | :------- | 
+| Expanding | Occurs when expanded |
+| Collapsed| Occurs when collapsed |
+
+
+
+# API Details
+
+~~~~
+enum ExpandDirection
+{
+    Down = 0,
+    Up = 1
+};
+
+runtimeclass ExpanderExpandingEventArgs
+{
+}
+
+runtimeclass ExpanderCollapsedEventArgs
+{
+}
+ 
+unsealed runtimeclass Expander : Windows.UI.Xaml.Controls.ContentControl
+{
+    Expander();
+
+    Object Header{ get; set; };
+    Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
+    Windows.UI.Xaml.Controls.DataTemplateSelector HeaderTemplateSelector{ get; set; };
+
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsExpanded{ get; set; };
+
+    [MUX_DEFAULT_VALUE("winrt::ExpandDirection::Down")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    ExpandDirection ExpandDirection{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderExpandingEventArgs> Expanding;
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderCollapsedEventArgs> Collapsed;
+
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateSelectorProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty IsExpandedProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ExpandDirectionProperty{ get; };
+}
+~~~~
+
+# Inputs and Accessibility
+
+## UI Automation Patterns
+Expander will use a ExpandCollapsePattern. Expanding/Collapsing the expander will raise [RaisePropertyChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisepropertychangedevent?view=winrt-19041) with the property changed being the [ExpandCollapseProperty](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.expandcollapsepatternidentifiers.expandcollapsestateproperty?view=winrt-19041) property.
+
+## Keyboard
+* Tabbing brings focus to the Header, neither the entire Expander nor the chevron is a tab-stop
+* When focus is on the Header, space key expands and collapse and focus does not move
+* Keyboard navigation inside the Expander's Header/Content is based on the content inside them
+
+## GamePad
+The Expander can be expanded and collapsed with A. Spatial navigation will navigate between the content of the Expander. 
+
+## Screen reader
+When focus is on the Header of an Expander, the screen reader will announce "Expander collapsed" or "Expander expanded". With focus staying on the header, the screen reader will then anounce "expanded" or "collapsed" accordingly. Expander should not have [focus engagement](https://docs.microsoft.com/en-us/windows/uwp/design/input/gamepad-and-remote-interactions#focus-engagement) enabled. 
```

Comment:
Does the screen reader also read the header? Is the header read first, and then "Expander collapsed/expanded" comes after?

Reply:
I think it's first the header content, then the collapsed/expanded property.